azure-pipelines.yml

```diff
@@ -0,0 +1,309 @@
+      reg query "HKLM\SOFTWARE\ODBC\odbcinst.ini\ODBC Driver 17 for SQL Server"
+      dir %WINDIR%\System32\msodbcsql*.dll
+    displayName: 'Install ODBC driver'
+    condition: false
+
+  # TOFIX: Install ODBC 13.1 because of SQLCMD 15 installation bug -- this step should be removed later
+  - script: msiexec /i "msodbcsql_13.1.msi" /q IACCEPTMSODBCSQLLICENSETERMS=YES ADDLOCAL=ALL
+    condition: false
+
+  # FOR SOME REASON the set up did not set the PATH 
+  - script: |
+      msiexec /i "MsSqlCmdLnUtils.msi" /qn IACCEPTMSSQLCMDLNUTILSLICENSETERMS=YES
+    displayName: 'Install SQL command line utilities version 15'
+    condition: false
+
+  - powershell: |
+      $client = New-Object Net.WebClient
+      $client.Headers.Add("user-agent", "azure pipeline build")
+      $client.DownloadFile("https://windows.php.net/downloads/releases/sha256sum.txt", "sha256sum.txt")
+      $env:VERSION=type sha256sum.txt | where { $_ -match "php-($(phpVersion)\.\d+)-src" } | foreach { $matches[1] }
+      Write-Host "Latest PHP $(phpVersion) is ${env:VERSION}" 
+      cd $(Build.SourcesDirectory)/buildscripts/
+      python builddrivers.py --PHPVER=${env:VERSION} --ARCH=x64 --THREAD=nts --SOURCE=$(Build.SourcesDirectory)/source --TESTING --NO_RENAME
+      cp php-sdk\phpdev\vc15\x64\php-${env:VERSION}-src\run-tests.php $(Build.SourcesDirectory)\test\functional\sqlsrv
+      cp php-sdk\phpdev\vc15\x64\php-${env:VERSION}-src\run-tests.php $(Build.SourcesDirectory)\test\functional\pdo_sqlsrv
+      dir *sqlsrv*.dll
+      cp *sqlsrv*.dll C:\tools\php\ext\
+    displayName: 'Build drivers for the latest version of PHP $(phpVersion)'
+  
+  - script: |
+      echo extension=php_sqlsrv.dll >> C:\tools\php\php.ini
+      echo extension=php_pdo_sqlsrv.dll >> C:\tools\php\php.ini
+      php --ri sqlsrv
+      php --ri pdo_sqlsrv
+    displayName: 'Load drivers'
+
+  - script: |
+      docker pull microsoft/mssql-server-windows-developer
+      docker run -d --name sqlcontainer -h $(host) -p 1433:1433 -e sa_password=$(pwd) -e ACCEPT_EULA=Y microsoft/mssql-server-windows-developer
+      docker ps -a
+    displayName: 'Run SQL Server for Windows Server'
+    condition: false
+
+  - script: | 
+      set path=C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;%path%
+      sqlcmd -S $(host) -U $(uid) -P $(pwd) -Q "SELECT @@Version"
+      set TEST_PHP_SQL_SERVER=$(host)
+      set TEST_PHP_SQL_UID=$(uid)
+      set TEST_PHP_SQL_PWD=$(pwd)
+      cd $(Build.SourcesDirectory)\test\functional\setup
+      python setup_dbs.py -dbname $(sqlsrv_db)
+      python setup_dbs.py -dbname $(pdo_sqlsrv_db)
+    displayName: 'Set up test databases'
+    condition: false
+
+  - script: |
+      cd $(Build.SourcesDirectory)\test\functional\sqlsrv
+      php run-tests.php -P sqlsrv_client_info.phpt
+      cd $(Build.SourcesDirectory)\test\functional\pdo_sqlsrv
+      php run-tests.php -P pdo_getAttribute_clientInfo.phpt

```

Comment:
Only running these two tests? Why are we going to the trouble of setting up test databases if we just do smoke testing?

Reply:
The smoke tests were there to make sure the extensions were loaded correctly. That being said, in Windows the conditions are mostly false, meaning to skip them. Those steps were kept for further improvements in the future

Reply:
ok that's fine.